src/tiles/tiles.service.ts

```diff
@@ -0,0 +1,60 @@
+import { Injectable, NotFoundException } from "@nestjs/common";
+import { Model } from "mongoose";
+import { InjectModel } from "@nestjs/mongoose";
+import { Tile } from "./schemas/tile.schema";
+import { CreateTileDto } from "./dto/create-tile.dto";
+import { UpdateTileDto } from "./dto/update-tile.dto";
+import { animationFrameScheduler } from "rxjs";
+
+@Injectable()
+export class TilesService {
+  constructor(@InjectModel(Tile.name) private readonly tileModel: Model<Tile>) {}
+  async findAll(): Promise<Tile[]> {
+    return await (await this.tileModel.find().exec()).filter((tile) => tile.isDeleted !== true);
+  }
+
+  async findOne(id: string): Promise<Tile> {
```

Comment:
If you wanna search tile by _id, I think the type of id should be ObjectId instead of string. I have checked your schema and I think you didn't add any other id for tile.